doc/source/options.rst

```diff
@@ -371,6 +371,9 @@ display.multi_sparse       True         "Sparsify" MultiIndex display (don't
 display.notebook_repr_html True         When True, IPython notebook will
                                         use html representation for
                                         pandas objects (if it is available).
+display.html_border        1            A ``border=value`` attribute is
```

Comment:
I would namespace this ``html``, as prob add more as time goes on.

Further would deprecate ``notebook_repr_html`` and rename to ``html.notebook`` (or *maybe* ``notebook.html=True``) (and not in the ``.display`` space to differentiate)